### PR TITLE
brave: Add beta and dev release channels

### PR DESCRIPTION
**Pull request by Velgus**

### CODE REVIEW

**Comment by linsui** [comment]
Please simplify the script. See https://github.com/lukesampson/scoop-extras/pull/2693/files

**Comment by Velgus** [comment]
Changed the existing Brave manifest to use the script as well.

File: bucket/brave-beta.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "bin": "brave.exe",
+    "shortcuts": [
+        [
+            "brave.exe",
+            "Brave Beta"
+        ]
+    ],
+    "checkver": {
+        "url": "https://github.com/brave/brave-browser/releases",
+        "regex": "Beta Channel v([\\d\\.]+)"
```
Comment (Ash258) [inline bucket/brave-beta.json]:
```suggestion
        "regex": "Beta Channel v([\\d.]+)"
```

File: bucket/brave-beta.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        },
+        "32bit": {
+            "url": "https://github.com/brave/brave-browser/releases/download/v0.69.115/brave_installer-ia32.exe#/dl.7z",
+            "hash": "9e670047348ee7ca88d3097715daa87d5d029d311f9a0ab7c312db84f8bdca6c"
+        }
+    },
+    "installer": {
+        "script": "Expand-7zipArchive \"$dir\\chrome.7z\" -ExtractDir 'Chrome-bin' -Removal"
+    },
+    "bin": "brave.exe",
```
Comment (Ash258) [inline bucket/brave-beta.json]:
```suggestion
    "bin": [
        [
            "brave.exe",
            "brave-beta"
        ]
   ],
```

File: bucket/brave-dev.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        },
+        "32bit": {
+            "url": "https://github.com/brave/brave-browser/releases/download/v0.70.74/brave_installer-ia32.exe#/dl.7z",
+            "hash": "a9cb3bdec385e5713f0ac756d16369b7024b6877c50ff4564f97a1c5a7313fa4"
+        }
+    },
+    "installer": {
+        "script": "Expand-7zipArchive \"$dir\\chrome.7z\" -ExtractDir 'Chrome-bin' -Removal"
+    },
+    "bin": "brave.exe",
```
Comment (Ash258) [inline bucket/brave-dev.json]:
```suggestion
    "bin": [
        [
            "brave.exe",
            "brave-dev"
        ]
   ],
```

File: bucket/brave-dev.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    "shortcuts": [
+        [
+            "brave.exe",
+            "Brave Dev"
+        ]
+    ],
+    "checkver": {
+        "url": "https://github.com/brave/brave-browser/releases",
+        "regex": "Dev Channel v([\\d\\.]+)"
```
Comment (Ash258) [inline bucket/brave-dev.json]:
```suggestion
        "regex": "Dev Channel v([\\d.]+)"
```

File: bucket/brave-nightly.json [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        },
+        "32bit": {
+            "url": "https://github.com/brave/brave-browser/releases/download/v0.71.35/brave_installer-ia32.exe#/dl.7z",
+            "hash": "b911efee1be433b3f9224614da4d80f5476a420678537f557767de64732529e4"
+        }
+    },
+    "installer": {
+        "script": "Expand-7zipArchive \"$dir\\chrome.7z\" -ExtractDir 'Chrome-bin' -Removal"
+    },
+    "bin": "brave.exe",
```
Comment (Ash258) [inline bucket/brave-nightly.json]:
```suggestion
    "bin": [
        [
            "brave.exe",
            "brave-nightly"
        ]
   ],
```

File: bucket/brave-nightly.json [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    "shortcuts": [
+        [
+            "brave.exe",
+            "Brave Nightly"
+        ]
+    ],
+    "checkver": {
+        "url": "https://github.com/brave/brave-browser/releases",
+        "regex": "Nightly Channel v([\\d\\.]+)"
```
Comment (Ash258) [inline bucket/brave-nightly.json]:
```suggestion
        "regex": "Nightly Channel v([\\d.]+)"
```

File: bucket/brave-beta.json [hunk 7]
```diff
@@ -0,0 +1,48 @@
+{
+    "homepage": "https://brave.com",
+    "version": "0.69.115",
+    "description": "Secure, Fast & Private Web Browser with Adblocker",
```
Comment (Ash258) [inline bucket/brave-beta.json]:
```suggestion
    "description": "Secure, Fast & Private Web Browser with Adblocker.",
```

**Comment by issaclin32** [comment]
This should be added to the *Versions* bucket.

**Comment by Velgus** [comment]
Shouldn't all the Firefox, Chrome, and Chromium manifests be in there too then?

**Comment by rashil2000** [comment]
Yes, we will migrate those soon

**Pull request closed without merging**